lib/bitgo/v1/api.rb

```diff
@@ -117,8 +121,30 @@ def delete_label(wallet_id: default_wallet_id, address:)
       # Wallets API
       ###############
 
-      def list_wallets
-        call :get, '/wallet'
+      # QUERY Parameters
+      # Parameter     Type     Required   Description
+      # enterpriseId  string   No         Filter list by Enterprise ID
+      # getbalances   boolean  No         Set to true to return the “balance” field for each wallet.
+      # limit         number   No         Max number of results to return in a single call (default=25, max=250)
+      # skip          number   No         The starting index number to list from. Default is 0.
+      def list_wallets(query: {})
+        call :get, '/wallet', query: query
+      end
+
+      # QUERY Parameters
+      # Parameter     Type     Required   Description
+      # skip          number   No         The starting index number to list from. Default is 0.
+      # limit         number   No         Max number of results to return in a single call (default=25, max=250)
+      # compact       boolean  No         Omit inputs and outputs in the transaction results
+      # minHeight     number   No         A lower limit of blockchain height at which the transaction was confirmed. Does not filter unconfirmed transactions.
+      # maxHeight     number   No         An upper limit of blockchain height at which the transaction was confirmed. Filters unconfirmed transactions if set.
+      # minConfirms   number   No         Only shows transactions with at least this many confirmations, filters transactions that have fewer confirmations.
+      def list_wallet_transactions(wallet_id: default_wallet_id, query: {})
+        call :get, "/wallet/#{wallet_id}/tx", query: query
+      end
+
+      def get_wallet_transaction(wallet_id: default_wallet_id, tx_id:)
```

Comment:
make tx_id a non-named required params as first argument
make the same for all other endpoints for required params